test/EFCore.SqlServer.FunctionalTests/Update/JsonUpdateSqlServerTest.cs

```diff
@@ -1328,20 +1328,52 @@ OUTPUT 1
 """);
     }
 
-    [ConditionalFact(Skip = "issue #30330")]
+    [ConditionalFact]
     public override async Task Edit_single_property_with_converter_string_True_False_to_bool()
     {
         await base.Edit_single_property_with_converter_string_True_False_to_bool();
 
-        AssertSql();
+        AssertSql(
+"""
+@p0='[false]' (Nullable = false) (Size = 7)
+@p1='1'
+
+SET IMPLICIT_TRANSACTIONS OFF;
+SET NOCOUNT ON;
+UPDATE [JsonEntitiesConverters] SET [Reference] = JSON_MODIFY([Reference], 'strict $.StringTrueFalseConvertedToBool', CAST(JSON_VALUE(@p0, '$[0]') AS bit))
+OUTPUT 1
+WHERE [Id] = @p1;
+""",
+                //
+                """
```

Comment:
Need to see if this can be enforced e.g. with .editorconfig

```suggestion
"""
```